src/mainwindow.cpp

```diff
@@ -658,7 +658,7 @@ void MainWindow::actAbout_triggered()
 {
      QMessageBox::about(this, tr("About"),
                      QStringLiteral("<center><b><big>QTerminal %1</big></b></center><br>").arg(qApp->applicationVersion())
-                     + tr("A lightweight and powerful multiplatform terminal emulator")
+                     + tr("A lightweight and powerful multiplatform terminal emulator (mouse swap build)")
```

Comment:
This feature will become part of the official version, so a cooment is no needed.